testnet/stacks-node/src/config.rs

```diff
@@ -2183,6 +2185,10 @@ pub struct MinerConfig {
     /// The minimum time to wait between mining blocks in milliseconds. The value must be greater than or equal to 1000 ms because if a block is mined
     /// within the same second as its parent, it will be rejected by the signers.
     pub min_time_between_blocks_ms: u64,
+    /// Time in milliseconds to pause after receiving the first rejection, before proposing a new block.
```

Comment:
Might be good ot say first "threshold rejection" specifically since its not just a single rejection that triggers a pause.

Reply:
Updated in 62470519f5d1ba99f0034030214e4089e04694d2